### DISCUSSION TITLE
Memory requirement

### DISCUSSION THREAD

**Discussion by Beiri22** [keystonejs/keystone#8189] [Questions]
I just installed the keystone.js demo project with just users and posts and managed to start it with pm2. As far as I can see, this nearly empty installation uses 800-900MB of memory (ram), which seems pretty much to me. Did I miss something or is keystone.js so memory hungry?

**Comment by flexdinesh** [comment]
Which demo project are you referring to @Beiri22?

**Reply by Beiri22** [reply]
The one you get when installing keystone.js via "npm init keystone-app@latest" ... there is a minimal schema given with two models: User and Post.

**Reply by flexdinesh** [reply]
800-900MB seems like a lot. Is that during build or when the server is running?

**Reply by Beiri22** [reply]
When running
